packages/core/src/runner/helper.ts

```diff
@@ -49,24 +49,21 @@ export async function assignLatestResult(updatedType: OutputType, result: Pipcoo
     break;
   case OutputType.Evaluate:
     console.log('evaluate result: ', result);
-    self.latestEvaluateResult = result as EvaluateResult;
-    if (self.latestEvaluateResult.pass === false) {
-      throw new EvaluateError(self.latestEvaluateResult);
+    self.evaluateMap = result as EvaluateResult;
+    if (self.evaluateMap.pass === true || self.evaluateMap.pass === false) {
+      self.evaluatePass = self.evaluateMap.pass;
+    }
+    if (self.evaluateMap.pass === false) {
+      throw new EvaluateError(self.evaluateMap);
     }
     break;
   case OutputType.ModelToSave:
     self.latestModel = result as UniModel;
-    if (saveModelCallback) {
-      const valueMap = 
-        (self.latestSampleData && self.latestSampleData.metadata 
-          && self.latestSampleData.metadata.label && self.latestSampleData.metadata.labelMap) || {};
-      fs.writeJSONSync(path.join(process.cwd(), '.temp', self.pipelineId, 'label.json'), valueMap);
-      await saveModelCallback(path.join(self.logDir, 'model'), self.pipelineId, path.join(process.cwd(), '.temp', self.pipelineId, 'label.json'));
-    } 
```

Comment:
Where to save the model now?

Reply:
The save model path and logic is not changed. This block is not used at all.